common/src/main/java/io/netty/util/concurrent/Future.java

```diff
@@ -83,12 +83,20 @@
     /**
      * Waits for this future until it is done, and rethrows the cause of the failure if this future
      * failed.
+     *
+     * @throws CancellationException if the computation was cancelled
+     * @throws {@link java.util.concurrent.CompletionException} if the computation threw an exception.
```

Comment:
Late, but this should be:

```
@throws CompletionException if the ....
```

Reply:
@trustin you mean removing the "package" ?